pxr/imaging/hdx/colorizeTask.cpp

```diff
@@ -419,6 +487,15 @@ static _Colorizer _colorizerTable[] = {
     { HdAovTokens->primId, HdFormatInt32, _colorizeId },
     { HdAovTokens->elementId, HdFormatInt32, _colorizeId },
     { HdAovTokens->instanceId, HdFormatInt32, _colorizeId },
+    { pxr::TfToken("primvars:st"), HdFormatFloat32Vec3, _colorizePrimvar }, // match _colorizeST for 3-channel st
+    // fallback converters
+    { HdPrimvarRoleTokens->none, HdFormatFloat32, _colorizeCameraDepth },
+    { HdPrimvarRoleTokens->none, HdFormatFloat32Vec2, _colorizeST },
+    { HdPrimvarRoleTokens->none, HdFormatFloat32Vec3, _float32Vec3ToDisplay },
+    { HdPrimvarRoleTokens->none, HdFormatFloat32Vec4, _float32ToDisplay },
```

Comment:
Here you are going to convert from linear to sRGB all unknown AOVs with HdFormatFloat32Vec{3/4} format. Have you considered non-color AOVs (e.g. world coordinate, velocity)?